pkg/etwlogrus/hook.go

```diff
@@ -58,6 +96,19 @@ func (h *Hook) Fire(e *logrus.Entry) error {
 		return nil
 	}
 
+	name := DefaultEventName
+	if h.getName != nil {
+		if n := h.getName(e); n != "" {
+			name = n
+		}
+	}
+
+	opts := make([]etw.EventOpt, 0, 3)
```

Comment:
Why do we initialize capacity to 3?

Reply:
I wanted to be generous and give users space for two extra options without having incur a realloc + copy

Reply:
A comment explaining would be helpful, since otherwise the inclination is to try and understand what is causing us to use 3 elements.

Reply:
Let's please add a little wording like "just in case the user passed a couple of options, so we don't need to reallocate quite as often" to the end